src/api/java/baritone/api/Settings.java

```diff
@@ -41,6 +41,12 @@
  */
 public final class Settings {
 
+
+    /**
+     * default farm radius. 0, is unlimited.
+     */
+    public final Setting<Integer> defaultFarmRange = new Setting<>(0);
```

Comment:
Do not put new settings at the top, near the top settings are ordered in rough order of importance. Put this down by other farm related settings, e.g. the nether wart one

Reply:
There is also an unnecessary new line :^)